### DISCUSSION TITLE
Google Cloud Provider?

### DISCUSSION THREAD

**Discussion by erikjanwestendorp** [SixLabors/ImageSharp.Web#377] [Ideas]
Are there any plans to add a provider for Google Cloud Storage?

**Comment by JimBobSquarePants** [comment]
Not currently since I've never used it however if someone wants to help build one, I'm happy to assist. We had the same situation with AWS.

**Reply by erikjanwestendorp** [reply]
@JimBobSquarePants Thanks for your reply, I would like to help build this 😄

**Reply by JimBobSquarePants** [reply]
Ace! Thanks! 

Have a look at the existing Azure and AWS implementations. They should be used as a pattern to follow. If you get stuck or simply need guidance please open a draft PR and I’ll help out.
